app/apollo/resolvers/channel.js

```diff
@@ -604,7 +573,7 @@ const channelResolvers = {
           subscriptionsRbacSync( [subscription], { resync: false }, context ).catch(function(){/*ignore*/});
         }
         catch( e ) {
-          logger.error(e, `${queryName} failed to create subscription '${subscription.name}' when serving ${req_id}.`);
+          logger.error(e, `${queryName} error creating subscription '${subscription.name}' when serving ${req_id}.`);
```

Comment:
Do we want to log an error here if this is an expected, continuable "exception?" Should be no worse than a warn, if logged at all.